src/OSPSuite.Core/Domain/Module.cs

```diff
@@ -26,8 +26,8 @@ protected Module(bool userEditable)
       public ISpatialStructure SpatialStructure { set; get; }
       public IObserverBuildingBlock ObserverBlock { set; get; }
       public IEventGroupBuildingBlock EventBlock { set; get; }
-      public IReadOnlyList<IMoleculeStartValuesBuildingBlock> MoleculeStartValueBlockCollection => _moleculeStartValueBlockCollection;
-      public IReadOnlyList<IParameterStartValuesBuildingBlock> ParametersStartValueBlockCollection => _parametersStartValueBlockCollection;
+      public IReadOnlyList<IMoleculeStartValuesBuildingBlock> MoleculeStartValuesBlockCollection => _moleculeStartValuesBlockCollection;
```

Comment:
Why do we have the BLock in the name. Can we remove it?

Reply:
Do you want to do that for all these? ObserverBlock => Observer?

Reply:
I'd say yes. Block does not mean anything in our context

Reply:
all done